dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -2494,6 +2508,117 @@ fn print_vcr(vcr: VolumeConstructionRequest, pad: usize) {
     }
 }
 
+enum VolumeLockHolder {
+    RegionReplacement { id: Uuid },
+    RegionSnapshotReplacement { id: Uuid },
+    RegionSnapshotReplacementStep { id: Uuid, request_id: Uuid },
+    UnknownMaybeSnapshot,
+}
+
+impl VolumeLockHolder {
+    pub fn type_string(&self) -> String {
+        match self {
+            VolumeLockHolder::RegionReplacement { .. } => {
+                String::from("region replacement")
+            }
+
+            VolumeLockHolder::RegionSnapshotReplacement { .. } => {
+                String::from("region snapshot replacement")
+            }
+
+            VolumeLockHolder::RegionSnapshotReplacementStep { .. } => {
+                String::from("region snapshot replacement step")
+            }
+
+            VolumeLockHolder::UnknownMaybeSnapshot => {
```

Comment:
why `UnknownMabyeSnapshot`?  Why not just `Unknown`?  I'm not sure if the maybe here is adding much.

What are the other possible choices?

Reply:
I think you're right, yeah - changed to unknown in 81c1e8718